azure-container-apps/README.md

```diff
@@ -83,7 +83,7 @@ After the deployment is complete, click **Go to resource**, then continue to ste
 
 ## Step 5: Test your SCIM bridge
 
-To test if your SCIM bridge is online, choose **Overview** in your application's sidebar, then click your **Application Url** link. This URL is your SCIM bridge domain. You should be able to enter your bearer token to verify that your SCIM bridge is up and running.
+To test if your SCIM bridge is online, choose **Overview** in your application's sidebar, then click your **Application Url** link. Thi is your **SCIM Bridge URL**. Sign in using your bearer token to verify that your SCIM Bridge is connected to your 1Password account.
```

Comment:
Small typo
```suggestion
To test if your SCIM bridge is online, choose **Overview** in your application's sidebar, then click your **Application Url** link. This is your **SCIM Bridge URL**. Sign in using your bearer token to verify that your SCIM Bridge is connected to your 1Password account.
```

Also SCIM bridge domain is what doc team is using on AKB instead of SCIM Bridge URL.

Reply:
I've chatted a bit with @Penderworth about this offline, but there's lots of precedent in our docs for things like "the URL for your SCIM Bridge" (even in [our style guide entry for 1Password SCIM Bridge](https://support.1password.com/style-guide/#1password-scim-bridge:~:text=The%20URL%20for%20your%20SCIM%20bridge.), funny enough).

I think there is a meaningful difference between them both that we should be careful about. The URL for a SCIM Bridge is composed of its domain and other important information:

SCIM Bridge URL: `https://scim.example.com/` (URL of public SCIM Bridge endpoint)
SCIM Bridge domain: `scim.example.com`
protocol: `https://`
port: `443` (implicitly here, but is also valid as part of the SCIM Bridge URL if serving on a different port, for example `https://scim.example.com:8443/`)
path: `/` (in above URL, but also `/app`, `/Users`, `/Groups`, etc.)

When setting a record name, there is not yet a URL. In the case of Google Workspace, we don't request or handle a full URL. In most other circumstances, a full URL is expected and required.

Reply:
Thanks for diving in on this and sharing your finds.

Reply:
I think that SCIM bridge URL is fine, but "bridge" should be lowercase.

Reply:
Noted:
```suggestion
To test if your SCIM bridge is online, choose **Overview** in your application's sidebar, then click your **Application Url** link. This is your **SCIM bridge URL**. Sign in using your bearer token to verify that your SCIM bridge is connected to your 1Password account.
```